www/flowjspixi.css

```diff
@@ -1,6 +1,6 @@
-*:not(body):not(html) {
+/* *:not(body):not(html) {
```

Comment:
We should have a probation period with this part deleted under 'new=1', before merging MMark.